src/main/java/ch/naviqore/raptor/router/Query.java

```diff
@@ -29,7 +29,7 @@ class Query {
     private final TimeType timeType;
 
     private final int[] targetStops;
-    private final int cutOffTime;
```

Comment:
Sorry, to be picky. but cut off is not one word 😅

Reply:
Sorry again, my bad. Turns out American English uses cutoff, just feels wrong (and I prefer British English).

Reply:
We should define what dialect to use and then apply this consistently throughout our codebase.

 Based on a short research (https://stackoverflow.com/questions/157807/gb-english-or-us-english), American English seems to be the preferred choice in programming.

There are also some British English that are kind of unusual for me:

| American English | British English |
|------------------|-----------------|
| Color            | Colour          |
| Initialize       | Initialise      |
| Optimize         | Optimise        |
| Center           | Centre          |
| License  | Licence  |
| Traveler         | Traveller       |
| Program          | Programme       |